### PR TITLE
rsync + raw_ssh by default, with gevent backend for celery

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

**Comment by dshulyak** [comment]
pushed image with tag - solarproject/solar-celery:gevent

File: solar/core/resource/virtual_resource.py [hunk 1]
```diff
@@ -293,12 +296,32 @@ def parse_dict_input(r_input, args):
...
+        assignments[r_input] = [arg]
+
+
 def is_connection(arg):
     if isinstance(arg, basestring) and '::' in arg:
         return True
     return False
 
 
+def parse_dicts_as_collection(child_input, arg):
```
Comment (loles) [inline solar/core/resource/virtual_resource.py]: We will have now `parse_dict_input` and `parse_dicts_as_collection`. Can you merge them?

File: examples/hosts_file/hosts.py [hunk 2]
```diff
@@ -10,31 +10,7 @@
 def run():
     ModelMeta.remove_all()
 
-    resources = vr.create('nodes', 'templates/nodes_with_transports.yaml', {'count': 2})
-    nodes = [x for x in resources if x.name.startswith('node')]
-    node1, node2 = nodes
-
-    hosts1 = vr.create('hosts_file1', 'resources/hosts_file', {})[0]
-    hosts2 = vr.create('hosts_file2', 'resources/hosts_file', {})[0]
-    node1.connect(hosts1, {
```
Comment (pigmej) [inline examples/hosts_file/hosts.py]: why it's removed ?

Comment (loles) [inline examples/hosts_file/hosts.py]: It's done in templates/nodes.yaml

Comment (pigmej) [inline examples/hosts_file/hosts.py]: Ah right ;)

Comment (dshulyak) [inline examples/hosts_file/hosts.py]: there is just too much of duplication + new transport

**Comment by pigmej** [comment]
gevent was not added in this PR.

**Pull request merged**